scripts/settings.js

```diff
@@ -1,108 +1,104 @@
-  var auto_archive= document.getElementById('auto-archive').checked;
-  var books = document.getElementById('books').checked;
-  var auto_update_context=document.getElementById('auto-update-context').checked;
-  var alexa= document.getElementById('alexa').checked;
-  var domaintools= document.getElementById('domaintools').checked;
-  var tweets= document.getElementById('tweets').checked;
-  var wbmsummary= document.getElementById('wbmsummary').checked;
-  var annotations= document.getElementById('annotations').checked;
-  var annotationsurl=document.getElementById('annotationsurl').checked;
-  var similarweb= document.getElementById('similarweb').checked;
-  var tagcloud = document.getElementById('tagcloud').checked;
-  var hoaxy = document.getElementById('hoaxy').checked;
-  var showall= document.getElementById('showall').checked;
-  var news = document.getElementById('news').checked;
-  var wikibooks = document.getElementById('wikibooks').checked;
-  var doi = document.getElementById('doi').checked;
+
+function save_options () {
+  var show_context = $('#show_context').val()
+  var auto_archive = $('#auto-archive').prop('checked')
+  var books = $('#books').prop('checked')
+  var auto_update_context = $('#auto-update-context').prop('checked')
+  var alexa = $('#alexa').prop('checked')
+  var domaintools = $('#domaintools').prop('checked')
+  var tweets = $('#tweets').prop('checked')
+  var wbmsummary = $('#wbmsummary').prop('checked')
+  var annotations = $('#annotations').prop('checked')
+  var annotationsurl = $('#annotationsurl').prop('checked')
+  var similarweb = $('#similarweb').prop('checked')
+  var tagcloud = $('#tagcloud').prop('checked')
+  var hoaxy = $('#hoaxy').prop('checked')
+  var showall = $('#showall').prop('checked')
+  var news = $('#news').prop('checked')
+  var wikibooks = $('#wikibooks').prop('checked')
+  var doi = $('#doi').prop('checked')
   chrome.storage.sync.set({
-    show_context:show_context,
+    show_context: show_context,
     auto_archive: auto_archive,
-    books:books,
-    auto_update_context:auto_update_context,
-    alexa:alexa,
-    domaintools:domaintools,
-    tweets:tweets,
-    wbmsummary:wbmsummary,
-    annotations:annotations,
-    annotationsurl:annotationsurl,
-    similarweb:similarweb,
-    tagcloud:tagcloud,
-    hoaxy:hoaxy,
-    showall:showall,
-    news:news,
-    wikibooks:wikibooks,
-    doi:doi
-  }, function() {
-    var status = document.getElementById('status');
-    status.textContent = 'Options saved.';
-    setTimeout(function() {
-      status.textContent = '';
-    }, 750);
-  });
+    books: books,
+    auto_update_context: auto_update_context,
+    alexa: alexa,
+    domaintools: domaintools,
+    tweets: tweets,
+    wbmsummary: wbmsummary,
+    annotations: annotations,
+    annotationsurl: annotationsurl,
+    similarweb: similarweb,
+    tagcloud: tagcloud,
+    hoaxy: hoaxy,
+    showall: showall,
+    news: news,
+    wikibooks: wikibooks,
+    doi: doi
```

Comment:
We don't need the intermediate variables for all settings. Could you please change it to be like:
```
chrome.storage.sync.set({
    show_context:  $('#show_context').val(),
    auto_archive: $('#auto-archive').prop('checked'),
    ...
});
```